src/SearchMachineProvider.jsx

```diff
@@ -1,16 +1,35 @@
 import { useMachine } from '@xstate/react';
-import { getItem, setItem } from 'localforage';
+import localforage from 'localforage';
 import PropTypes from 'prop-types';
 import { createContext, useContext } from 'react';
 import { assign, createMachine } from 'xstate';
 import { downloadFormats, fieldNames } from '../functions/common/config';
+import stateOfUtahJson from './data/state-of-utah.json';
+
+localforage.config({
+  name: 'deq-enviro-search-cache',
+});
+
+const VERSION_KEY = 'searchContextVersion';
+const CURRENT_VERSION = 1; // change this anytime you change the schema of the cache (includes any layer config changes)
```

Comment:
**suggestion**: do something with CI since this will probably be hard to remember after some time.

Reply:
I thought about this but wasn't sure how I would trigger the change. It would have to change any time certain lines changed in the file changed. Maybe I should just change it each time I cut a new release? Hmmm....